etna/articles/models.py

```diff
@@ -376,6 +374,7 @@ class Meta:
             index.SearchField("body"),
             index.SearchField("topic_names", boost=1),
             index.SearchField("time_period_names", boost=1),
+            index.SearchField("author_name", boost=1),
```

Comment:
You might want to consider using [index.RelatedFields](https://docs.wagtail.org/en/stable/topics/search/indexing.html#index-relatedfields) here instead.